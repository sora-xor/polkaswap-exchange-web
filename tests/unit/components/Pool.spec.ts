// import { shallowMount, createLocalVue } from '@vue/test-utils'
// import Vuex from 'vuex'

// import Pool from '@/components/Pool.vue'
// import { tokens } from '../../mocks/tokens'
// import { SoramitsuElementsImport, TranslationMock } from '../../utils'

// const localVue = createLocalVue()
// localVue.use(Vuex)
// SoramitsuElementsImport(localVue)

// describe('Pool.vue', () => {
//   let actions
//   let getters
//   let store

//   beforeEach(() => {
//     TranslationMock(Pool)

//     actions = {
//       getLiquidity: jest.fn()
//     }

//     getters = {
//       liquidity: () => false,
//       firstToken: () => tokens[0],
//       secondToken: () => tokens[1],
//       firstTokenValue: () => 100,
//       secondTokenValue: () => 45.4545
//     }

//     store = new Vuex.Store({
//       actions,
//       getters
//     })
//   })

//   it('should renders correctly', () => {
//     const wrapper = shallowMount(Pool, { localVue, store })
//     expect(wrapper.element).toMatchSnapshot()
//   })
// })

describe('Pool test', () => {
  test('Temporary test', () => {
    expect(true).toEqual(true);
  });
});
