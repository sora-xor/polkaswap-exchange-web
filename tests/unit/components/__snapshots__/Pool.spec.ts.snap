// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pool.vue should renders correctly 1`] = `
<div
  class="el-form--pool"
>
  <div
    class="header"
  >
    <h3
      class="header-title"
    >
      
    </h3>
     
    <s-tooltip-stub
      animation="el-fade-in-linear"
      borderradius="mini"
      class="header-tooltip"
      closedelay="0"
      content="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."
      offset="0"
      opendelay="0"
      placement="bottom-end"
      popperclass="info-tooltip"
      tabindex="0"
      theme="light"
    >
      <s-icon-stub
        name="info"
        size="16"
      />
    </s-tooltip-stub>
  </div>
   
  <p
    class="pool-info-container"
  >
    
    
  
  </p>
   
  <s-collapse-stub
    borders="true"
    class="pool-info-container"
  />
   
  <s-button-stub
    borderradius="small"
    class="el-button--add-liquidity"
    icon=""
    nativetype="button"
    size="medium"
    tabindex="0"
    tooltip=""
    tooltipplacement="top"
    type="primary"
  >
    
    
  
  </s-button-stub>
   
  <!---->
</div>
`;
